montblanc/examples/MS_tf_example.py

```diff
@@ -107,9 +112,14 @@ def model_vis(self, context):
         type=lambda v: v.lower() in ("yes", "true", "t", "1"),
         choices=[True, False], default=False,
         help='Handle auto-correlations')
+    parser.add_argument('-nt','--numthreads',dest='nt',
+        type=int, default=1, help='Number of Tensorflow threads')
```

Comment:
Can this be removed? It looks like its unused elsewhere. Maybe debugging code?

Reply:
done